src/python/pants/backend/BUILD

```diff
@@ -38,6 +38,7 @@ __dependents_rules__(
             "[/python/lint/yapf/rules.py]",
             "[/python/lint/yapf/subsystem.py]",
             "[src/python/pants/backend/python/goals/lockfile.py]",
+            "[src/python/pants/backend/python/subsystems/python_tool_base.py]",
```

Comment:
So that `build_files/fmt/{black/yapf}` can import get_lockfile_interpreter_constraints from python_tool_base.py.